src/sharing/ShareToStream.js

```diff
@@ -150,64 +94,45 @@ class ShareToStream extends React.Component<Props, State> {
 
   render() {
     const { sharedData } = this.props.route.params;
-    const { stream, topic, message, isStreamFocused, isTopicFocused, sending } = this.state;
+    const { stream, topic, isStreamFocused, isTopicFocused } = this.state;
     const narrow = streamNarrow(stream);
+    const sendTo = { stream, topic, type: 'stream' };
 
     return (
-      <>
-        <ScrollView style={styles.wrapper} keyboardShouldPersistTaps="always" nestedScrollEnabled>
```

Comment:
it is necessary. Fixed in the new revision.

Reply:
Cool. The new revision sets that prop in the new wrapper.

That does still make this change not [NFC](https://github.com/zulip/zulip-mobile/blob/master/docs/glossary.md#nfc), because previously it wasn't set for `ShareToPm` and now it is. Two ways to fix that:
* The cleanest is to have a small commit first that adds it on `ShareToPm` and says why. Then the main commit can be NFC.
  * In this case (https://github.com/zulip/zulip-mobile/pull/4757#issuecomment-881039196), it's OK if the "why" doesn't involve really investigating how or whether the old code worked -- if the story is "we want it for the stream case, and it seems fine here", that's enough.
* Alternatively the main commit could drop the "[nfc]", and mention this change in the commit message. That wouldn't normally meet our standards for highly readable commits, but [in this case](https://github.com/zulip/zulip-mobile/pull/4757#issuecomment-881039196) it'd be OK.

I think the cleaner thing is actually pretty quick to do, though, with the right tools. (Much easier than splitting up the bigger changes in this commit into several steps would be.) So I'd encourage you to try it. I'd use `git rebase -i`, with something like:
```
$ gitk --all HEAD &  # to see what you're doing
$ git rebase -i
  # insert a 'break' line after this "Deduplicate code" commit
  # save and exit; now you're mid-rebase, and at this commit
$ git reset --hard @~  # reset to before this commit
  # now edit ShareToPm.js to change that one line; save that
$ git commit -am 'wip add nestedScrollEnabled'
$ git checkout @{2} .  # get to the same state as after this commit
$ git commit -aC @{2}  # borrow commit message, and author name+date, from this commit
$ git rebase --continue
```

The two references to `@{2}` there are a way of mentioning the "Deduplicate code" commit -- `@{0}` is the current HEAD, `@{1}` is what it was before this (before the `git commit`), and `@{2}` is what it was before that (before the `git reset`). Alternatively, you could look at the `gitk --all HEAD` or a `git log` command, and copy-paste the commit ID.

Then after that:
* refresh in gitk, to check your work;
* use `git rebase -i` and just a `reword` on the new commit, to replace the placeholder message with a proper one.